nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/AbstractExecuteSQL.java

```diff
@@ -82,6 +84,16 @@
             .identifiesControllerService(DBCPService.class)
             .build();
 
+    public static final PropertyDescriptor SQL_PRE_QUERY = new PropertyDescriptor.Builder()
+            .name("sql-pre-query")
+            .displayName("SQL pre-query")
```

Comment:
Can you make this `SQL Pre-Query`? Same for `SQL Post-Query`.

Reply:
Hi Peter, thanks for the feedback.
I intentionally type it in 'SQL pre-query' to align with SQL_SELECT_QUERY property displayName 'SQL select query'.

For example currently,
SQL_PRE_QUERY, display '**SQL pre-query**'
SQL_SELECT_QUERY, display '**SQL select query**'
SQL_POST_QUERY, display '**SQL post-query**'

If modified as you mention (I actually try this version in the first try)
For example,
SQL_PRE_QUERY, display '**SQL Pre-Query**'
SQL_SELECT_QUERY, display '**SQL select query**'
SQL_POST_QUERY, display '**SQL Post-Query**'

Please let me know your though on this alignment, thanks

Reply:
They are capitalized in the HIVE processors, so this would match them.